### PR TITLE
update node version in request to latest

### PR DESCRIPTION
**Pull request by ghost**
request was using node version 10.13.0 which has long stopped receiving Security Support.

### CODE REVIEW

**Review by oilyquack — APPROVED** [review]
Did you also try bumping to version 18? If that's possible without any issue that saves us a job come September when v16 is end of life.

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "serve": "^11.3.0"
   },
   "engines": {
-    "node": ">= 10.13.0",
+    "node": "^16.0.0",
```
Comment (simon360) [inline package.json]: I think we'll want to use
```suggestion
    "node": ">= 16.0.0",
```
here.

`^16.0.0` will mean this package can only be installed on Node v16. In other words: it'll raise an error (or warning, at least) if we install it on a project on Node 18.

But `>= 16.0.0` will mark it as working when Node is _at least_ v16, which is closer to what we want.

(I wish JSON had support for comments so we could document the reasoning here!)

**Review by simon360 — DISMISSED** [review]

**Comment by ghost** [comment]
> Did you also try bumping to version 18? If that's possible without any issue that saves us a job come September when v16 is end of life.

@simon360 left a comment in the ticket for this which I was going off- "As such, we will want to make sure that we set this field to the earliest version of Node we continue to support. I'd suggest setting it to 16."
But happy to go with the consensus. @oilyquack

**Comment by oilyquack** [comment]
> > Did you also try bumping to version 18? If that's possible without any issue that saves us a job come September when v16 is end of life.
> 
> @simon360 left a comment in the ticket for this which I was going off- "As such, we will want to make sure that we set this field to the earliest version of Node we continue to support. I'd suggest setting it to 16." But happy to go with the consensus. @oilyquack

I see, makes sense. Ignore my comment then!

File: package.json (hunk 1 above)
Comment (ghost) [reply package.json]: I'll make the commit message clear and hopefully that will help explain

**Pull request merged**